src/sdltiles.cpp

```diff
@@ -1306,6 +1319,7 @@ void CheckMessages()
                 quit = true;
                 break;
         }
+        if (text_reflesh) break;
```

Comment:
Styling:
```
if( x ) {
    // Stuff
}
```

Braces shouldn't be skipped, parentheses should be padded on the inside, no space between `if` and parentheses.

Reply:
Also, looks like a typo